load-testing/tests/relays_stress_test.go

```diff
@@ -453,9 +455,12 @@ func (s *relaysSuite) TheCorrectPairsCountOfClaimAndProofMessagesShouldBeCommitt
 		s.currentProofCount,
 		"claims and proofs count mismatch",
 	)
-	require.Equal(s,
-		s.expectedClaimsAndProofsCount,
-		s.currentProofCount,
-		"unexpected claims and proofs count",
-	)
+	// TODO_TECHDEBT: The current counting mechanism for the expected claims and proofs
```

Comment:
How much of a concern is this?

Can't tell if it's "hard to do" or a major issue?

Reply:
Since we don't have full control from within the test over how `Gateways` distribute their relays or info about which `Supplier` served a given request.

Although it's possible to do it with some modifications on the `AppGateServer`, results and failed proofs would anyway be visible on grafana, so I don't believe it to be a big concern.